rotkehlchen/api/v1/parser.py

```diff
@@ -96,6 +101,17 @@ def parse(  # type: ignore  # we have added the resource_object on top of parse
             location_data = self._load_location_data(
                 schema=schema, req=req, location=location,
             )
+            if allow_async_validation and location_data.get('async_query', False) is True:
```

Comment:
What exactly is `location_data`? I thought `data`is where the request args live. Why do you need to check this and not pure `data` from `schema.load()` down there?

Reply:
location_data is the raw json that the frontend sends. The reason to not check `data` later is because at that moment we have called `.load` adn the `post_load` action has been triggered and that one is making the ens queries

Reply:
can you also add this to the comment above. Explain what the location data is and why they are used?